### DISCUSSION TITLE
Add tyre age of the best time to the tyre window

### DISCUSSION THREAD

**Discussion by pyth0on** [f1multiviewer/issue-tracker#295] [Feature requests]
Would be nice to add the tyre age behind the lap of the best time. (e.g. 4 (Age 10)).
![image](https://user-images.githubusercontent.com/65238629/201483549-b9b6109f-de0f-4816-ac0a-fca1ac45c2e0.png)

**Comment by LindaJuffermans** [comment]
Possibly related: In FP show what tyre was used to set the (current) fastest lap for a driver.

<img width="1892" height="222" alt="image" src="https://github.com/user-attachments/assets/64d073e1-ec89-4193-a585-f250ddc7b4e2" />
